BitwardenShared/UI/Tools/Generator/Generator/GeneratorProcessor.swift

```diff
@@ -312,6 +313,20 @@ final class GeneratorProcessor: StateProcessor<GeneratorState, GeneratorAction,
         didLoadGeneratorOptions = true
     }
 
+    /// Re-loads generator options if needed, avoiding duplicate calls on first load.
+    ///
+    private func reloadGeneratorOptionsIfNeeded() async {
+        if state.isFirstLoad {
+            state.isFirstLoad = false
+        } else {
+            do {
+                try await loadGeneratorOptions()
+            } catch {
+                services.errorReporter.log(error: error)
+            }
+        }
+    }
+
```

Comment:
⛏️ Use `guard` to avoid nesting:
```suggestion
    private func reloadGeneratorOptionsIfNeeded() async {
        guard !state.isFirstLoad else {
            state.isFirstLoad = false
            return
        }

        do {
            try await loadGeneratorOptions()
        } catch {
            services.errorReporter.log(error: error)
        }
    }

```